src/ngraph/runtime/tensor.cpp

```diff
@@ -70,3 +70,10 @@ void runtime::Tensor::set_stale(bool val)
 {
     m_stale = val;
 }
+
+void runtime::Tensor::copy_to(shared_ptr<runtime::Tensor> d, size_t offset, size_t n)
+{
+    void* copy = malloc(n);
```

Comment:
You should use `unique_ptr<int64_t[]>` so that you get automatic memory freeing even if an exception occurrs somewhere and you get better memory alignment with int16_t which must be on 8 byte boundary